src/index.js

```diff
@@ -14,15 +15,11 @@ const {
   style, sub, sup, table, tbody, td, textarea, tfoot, th,
   thead, title, tr, u, ul, video,
 } = require(`hyperscript-helpers`)(h)
-let matchesSelector
-try {
-  matchesSelector = require(`matches-selector`)
-} catch (e) {
-  matchesSelector = () => {}
-}
-import fastMap from 'fast.js/array/map'
-import reduce from 'fast.js/array/reduce'
+import matchesSelector from 'snabbdom-selector'
 import filter from 'fast.js/array/filter'
+import reduce from 'fast.js/array/reduce'
+import concat from 'fast.js/array/concat'
+import fastMap from 'fast.js/array/map'
```

Comment:
I was only trying to avoid importing anything more than needed. I have no objections if that would be preferred?